.github/workflows/automerge.yml

```diff
@@ -10,34 +10,67 @@ jobs:
     name: Merge Master to Staging
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
       - name: Set Git config
         run: |
           git config --local user.email "Sofie-Automation@users.noreply.github.com"
           git config --local user.name "GitHub Actions # Sofie-Automation"
       - name: Merge 'master' to 'staging'
+        id: Merge
```

Comment:
I would like this to be lowercase snake_case `merge`. The same goes for the next staging->develop job.